vector/src/main/java/im/vector/app/core/extensions/Context.kt

```diff
@@ -17,9 +17,32 @@
 package im.vector.app.core.extensions
 
 import android.content.Context
+import android.graphics.drawable.Drawable
+import androidx.annotation.ColorInt
+import androidx.annotation.ColorRes
+import androidx.annotation.DrawableRes
+import androidx.core.content.ContextCompat
 import dagger.hilt.EntryPoints
 import im.vector.app.core.di.SingletonEntryPoint
+import kotlin.math.round
 
 fun Context.singletonEntryPoint(): SingletonEntryPoint {
     return EntryPoints.get(applicationContext, SingletonEntryPoint::class.java)
 }
+
+fun Context.getResTintedDrawable(@DrawableRes drawableRes: Int, @ColorRes tint: Int, alpha: Float? = null): Drawable? {
```

Comment:
Nit: you could annotate the param `alpha` with `@FloatRange(from = 0.0, to = 1.0)`
Nit2: maybe defaulting the param `alpha`to the value `1.0` to avoid making it nullable?

Reply:
https://github.com/vector-im/element-android/pull/5025/commits/f8e65f5872054cf277e66c298dfc44f6b792d501